SU-SWS/linky_clicky
# Update feature for Services

# READY FOR REVIEW

# Summary
- An updated Feature for `services`

# Needed By (Date)
- Whenever

# Criticality
- How critical is this PR on a 1-10 scale? 2; not very critical

# Steps to Test

1. Check out this branch
2. Run `behat features/low-value/services.feature`
3. Verify that it passes
4. See whether it hangs for you at the end. Not sure if it's something with the Feature or something with my local config.

# Affected Projects or Products
- Sites

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-99
- Other info: After much trial and error, I could not get a RESTful Behat extension working, so I decided to punt and just check for a 200 response on the JSON endpoint.
